snapshotter_opts_windows.go

```diff
@@ -0,0 +1,30 @@
+//go:build windows
+// +build windows
```

Comment:
Not needed, as the filename already has the `_windows` suffix

Reply:
@dgl Wanna fix this? :)

Reply:
Done.